history/taskana-simplehistory-provider/src/test/java/acceptance/query/QueryHistoryAccTest.java

```diff
@@ -25,19 +25,19 @@ public QueryHistoryAccTest() {
   @Test
   public void testListValuesAscendingAndDescending() {
```

Comment:
could you change the name of the tests to the new schema, please?

Reply:
Done